index.html

```diff
@@ -11,21 +11,31 @@
   <body>
     <div class="page-preview">
       <nav>
-        <div class="left-logo">Gordon Otieno</div>
-        <div class="humburger"><img src="./icons/hamburger.png" alt="logo"></div>
+        <div class="left-logo"><a href="#">Gordon Otieno</a></div>
+        <div class="humburger"><img src="./icons/Icon-Menu.png" alt="logo"></div>
       </nav>
       <div class="intro">
         <h1>Hay There I'm Gordon</h1>
         <h2>I am Software Developer</h2>
         <p>I can help you develop develop application, Build websites, provide you with hosting servises and many more.
            Please feel free to contact me on my social media accounts below</p>
         </div>
-         <div class="social-media">
-          <div class="linkedin-icon"><img src="/icons/github.png" alt="linkedin"></div>
-          <div class="angelist-icon"><img src="/icons/angelist.png" alt="angelist"></div>
-          <div class="twiter-icon"><img src="/icons/twiter.png" alt="twitter"></div>
-          <div class="medium-icon"><img src="/icons/medium.png" alt="medium"></div>
-        </div>
+        <ul class="social-media">
+          <li class="linkedin-icon">
+            <a href="#"><img src="./icons/github.png" alt="linkedin"></a>
+          </li>
+          <li class="angelist-icon">
+            <a href="#"><img src="./icons/angel-list.png" alt="angelist"></a>
+          </li>
+          <li class="twiter-icon">
+            <a href="#"><img src="./icons/twitter.png" alt="twitter"></a>
+          </li>
+          <li class="medium-icon">
+            <a href="#"><img src="./icons/medium.png" alt="medium"></a>
+          </li>
```

Comment:
- [ ]  You should use the same icons similar to the one from Figma design. 
Your Design 
![image](https://user-images.githubusercontent.com/43877689/176653669-dacb7583-4dc7-4cbf-8880-4a2c74d10936.png)
Figma design 
![image](https://user-images.githubusercontent.com/43877689/176653995-de24308d-09ec-4265-98fe-d11e632be7aa.png)